xml/System.Xml.Linq/XElement.xml

```diff
@@ -3841,7 +3839,7 @@ orderDate=10/6/2006
 ## Remarks  
  This conversion operator uses the <xref:System.Xml.XmlConvert> class to do the conversion.  
   
- The equivalent method for this operator is [Explicit Conversion (XElement to](<!-- TODO: review code entity reference <xref:assetId:///op_Explicit(XAttribute>  --> attribute)?qualifyHint=True&autoUpgrade=False)  
+ The equivalent method for this operator is <xref:System.Xml.XmlConvert.ToDateTimeOffset(System.String)?displayProperty=fullName>. 
```

Comment:
Here again, this is element.Value, not the element itself, that has to be passed to the conversion method.